frontend/app/components/menu.tsx

```diff
@@ -37,7 +41,7 @@ export function Menu({ className, children }: MenuProps) {
         <span id="menu-label">{t('gcweb:app.menu')}</span>
         <DownChevron />
       </DropdownMenuTrigger>
-      <DropdownMenuContent align="start" className="w-64">
+      <DropdownMenuContent align="start" className="w-64 !bg-slate-700">
```

Comment:
No need to use `!important` since `DropdownMenuContent` uses `cn` Tailwind utility function and will merge Tailwind CSS classes without style conflicts.

https://github.com/dcastil/tailwind-merge

```suggestion
      <DropdownMenuContent align="start" className="w-64 bg-slate-700">
```